jobs/checkopenhabsoffline.js

```diff
@@ -1,64 +1,67 @@
-    Openhab.find({status:'offline', last_online: {"$lt":date3DaysAgo}}, function(error, openhabs) {
-        if (!error && openhabs) {
-            logger.info("openHAB-cloud: Found " + openhabs.length + " openhabs");
-    //        console.log(openhabs);
-            for (var i in openhabs) {
-                var openhab = openhabs[i];
-                if (!openhab.last_email_notification || openhab.last_email_notification < date3DaysAgo) {
-                    openhab.last_email_notification = new Date;
-                    openhab.save();
-//                    console.log(openhab.uuid + ":");
-                    UserAccount.findOne({_id:openhab.account}, function(error, userAccount) {
-                        if (!error && userAccount) {
-                            User.find({account: userAccount.id, role:"master"}, function(error, users) {
-                                if (!error && users) {
-                                    for (var i in users) {
-                                        var user = users[i];
-                                        var locals = {
-                                            email: user.username
-                                        };
-                                        mailer.sendEmail(user.username, "We are worried about your openHAB",
-                                            "openhaboffline", locals, function(error) {
-                                                if (!error) {
-//                                                    logger.info("mail sent");
-                                                } else
-                                                    logger.error("openHAB-cloud: Error sending email: " + error);
-                                            });
-                                    }
-                                }
-                            });
-                        } else if (error) {
-                            logger.error("openHAB-cloud: Error finding user account for openhab: " + error);
-                        } else {
-                            logger.error("openHAB-cloud: Unable to find user account for openhab which is nonsense");
-                        }
-                    });
-                }
+    logger.info('openHAB-cloud: date3DaysAgo = ' + date3DaysAgo);
+    Openhab.find({status:'offline', last_online: {'$lt':date3DaysAgo}}, function (error, openhabs) {
+        if (error) {
+            logger.error('openHAB-cloud: Error finding offline openHABs: ' + error);
+        }
+
+        if (!openhabs) {
+            logger.info('openHAB-cloud: No offline openHABs found');
+        }
+        logger.info('openHAB-cloud: Found ' + openhabs.length + ' openhabs');
+        for (var i in openhabs) {
+            var openhab = openhabs[i];
+
+            if (openhab.last_email_notification && openhab.last_email_notification > date3DaysAgo) {
+                continue;
```

Comment:
@FlorianSW wow great! The code is much more readable and cleaner now!
Great job! I will deep more into this after merging your other PR.